lib/react_client/react_interop.dart

```diff
@@ -152,8 +152,6 @@ class ReactComponent {
   external get refs;
   external void setState(state, [callback]);
   external void forceUpdate([callback]);
-
-  external bool isMounted();
```

Comment:
Reminder: we need to deprecate this in the current master line using a `@Deprecated('5.0.0')` annotation.